.gitignore

```diff
@@ -1,2 +1,3 @@
 /target/
 Cargo.lock
+.idea/
```

Comment:
This .gitignore change doesn't belong in a feature PR

Reply:
just a tip: you can put all your IDE-related ignores in `.git/info/exclude`